src/tests/Microsoft.PowerFx.Connectors.Tests/PowerAppsForMakersTests.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.OpenApi.Models;
+using Microsoft.PowerFx.Functions;
+using Microsoft.PowerFx.Tests;
+using Microsoft.PowerFx.Types;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.PowerFx.Connectors.Tests
+{
+    public class PowerAppsForMakersTests : BaseConnectorTest
+    {
+        public PowerAppsForMakersTests(ITestOutputHelper output)
+            : base(output, @"Swagger\PowerAppsForMakers.json")
+        {
+        }
+
+        public override string GetNamespace() => "PowerAppsForMakers";
+
+        public override string GetEnvironment() => "a323f485-54aa-e556-937d-86727a0c5ac0";
+
+        public override string GetEndpoint() => "tip1002-002.azure-apihub.net";
+
+        public override string GetConnectionId() => "763ee25d4f4046debbfcddcf9175db97";
+
+        public override string GetJWTToken() => "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6IjVCM25SeHRRN2ppOGVORGMzRnkwNUtmOTdaRSIsImtpZCI6IjVCM25SeHRRN2ppOGVORGMzRnkwNUtmOTdaRSJ9.eyJhdWQiOiJodHRwczovL2FwaWh1Yi5henVyZS5jb20iLCJpc3MiOiJodHRwczovL3N0cy53aW5kb3dzLm5ldC80YWJjMjRlYS0yZDBiLTQwMTEtODdkNC0zZGUzMmNhMWU5Y2MvIiwiaWF0IjoxNzA0ODIxOTE2LCJuYmYiOjE3MDQ4MjE5MTYsImV4cCI6MTcwNDgyNjc1OCwiYWNyIjoiMSIsImFpbyI6IkFUUUF5LzhWQUFBQUNlUXdzMHZIWnp2enN3VEkxMDJEdUFFZzRkZ2hibTNnRDFiOGFPN3BKSDlJUFlXYStWaXVId1pXYng3VEZDNFAiLCJhbXIiOlsicHdkIl0sImFwcGlkIjoiYThmN2E2NWMtZjViYS00ODU5LWIyZDYtZGY3NzJjMjY0ZTlkIiwiYXBwaWRhY3IiOiIwIiwiZmFtaWx5X25hbWUiOiJhdXJvcmEiLCJnaXZlbl9uYW1lIjoidXNlcjAxIiwiaXBhZGRyIjoiOTAuMTA0LjczLjIwMyIsIm5hbWUiOiJhdXJvcmF1c2VyMDEiLCJvaWQiOiJkOGU5MGRiNi1kMDljLTRhOTAtYmMxZS04NjUxMmE3OTdmZDMiLCJwdWlkIjoiMTAwMzIwMDJCQTBENzQ1NyIsInJoIjoiMC5BYmNBNmlTOFNnc3RFVUNIMUQzakxLSHB6Rjg4QmY2U05oUlBydkx1TlB3SUhLNjNBQnMuIiwic2NwIjoiUnVudGltZS5BbGwiLCJzdWIiOiJONlRHR2FmckFfZVVOa3JtUnZoLXdBYlp2SXFEamg5YVFYZEVaN1BZS29JIiwidGlkIjoiNGFiYzI0ZWEtMmQwYi00MDExLTg3ZDQtM2RlMzJjYTFlOWNjIiwidW5pcXVlX25hbWUiOiJhdXJvcmF1c2VyMDFAYXVyb3JhZmluYW5jZWludGVncmF0aW9uMDIub25taWNyb3NvZnQuY29tIiwidXBuIjoiYXVyb3JhdXNlcjAxQGF1cm9yYWZpbmFuY2VpbnRlZ3JhdGlvbjAyLm9ubWljcm9zb2Z0LmNvbSIsInV0aSI6ImFZMkdwM18xVWstRjRxRmtsaFFUQkEiLCJ2ZXIiOiIxLjAifQ.O-zTvqQfebyBfkX-ksbN95_2RwFBd062tRA52IB3orczQ5ub9xvI-VWbUlwkwzP3OBU4p4JdQdBM8R6Wc0UozK2H8K0b5ReEpLy2OS2ynEQnynuYht4MiBdz6AfcQ0TweILgA8FOYkdkSL0k9ZwCT4XPVbKLDjGadyI97UFvS20XxkEC30SDQQR7AtBa4qyWTfd9QePLAeuMvcghFaM7zvUHQZbNikm5AiiSTHP2map181AhmYlfO7Mv5q7sMb4mxkdIrzhnC64mwViuSijByebgs3MIQdmHJLF6hQY_NaKKExfPMKNZcViVKS_mCKKTQe-MN3SkF5wAKbW-7xGi6A";
```

Comment:
>eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6IjVCM25SeHRRN2ppOGVORGMzRnkwNUtmOTdaRSIsImtpZCI6IjVCM25SeHRRN2ppOGVORGMzRnkwNUtmOTdaRSJ9.eyJhdWQiOiJodHRwczovL2FwaWh1Yi5henVyZS5jb20iLCJpc3MiOiJodHRwczovL3N0cy53aW5kb3dzLm5ldC80YWJjMjRlYS0yZDBiLTQwMTEtODdkNC0zZGUzMmNhMWU5Y2MvIiwiaWF0IjoxNzA0ODIxOTE2LCJuYmYiOjE3MDQ4MjE5MTYsImV4cCI6MTcwNDgyNjc1OCwiYWNyIjoiMSIsImFpbyI6IkFUUUF5LzhWQUFBQUNlUXdzMHZIWnp2enN3VEkxMDJEdUFFZzRkZ2hibTNnRDFiOGFPN3BKSDlJUFlXYStWaXVId1pXYng3VEZDNFAiLCJhbXIiOlsicHdkIl0sImFwcGlkIjoiYThmN2E2NWMtZjViYS00ODU5LWIyZDYtZGY3NzJjMjY0ZTlkIiwiYXBwaWRhY3IiOiIwIiwiZmFtaWx5X25hbWUiOiJhdXJvcmEiLCJnaXZlbl9uYW1lIjoidXNlcjAxIiwiaXBhZGRyIjoiOTAuMTA0LjczLjIwMyIsIm5hbWUiOiJhdXJvcmF1c2VyMDEiLCJvaWQiOiJkOGU5MGRiNi1kMDljLTRhOTAtYmMxZS04NjUxMmE3OTdmZDMiLCJwdWlkIjoiMTAwMzIwMDJCQTBENzQ1NyIsInJoIjoiMC5BYmNBNmlTOFNnc3RFVUNIMUQzakxLSHB6Rjg4QmY2U05oUlBydkx1TlB3SUhLNjNBQnMuIiwic2NwIjoiUnVudGltZS5BbGwiLCJzdWIiOiJONlRHR2FmckFfZVVOa3JtUnZoLXdBYlp2SXFEamg5YVFYZEVaN1BZS29JIiwidGlkIjoiNGFiYzI0ZWEtMmQwYi00MDExLTg3ZDQtM2RlMzJjYTFlOWNjIiwidW5pcXVlX25hbWUiOiJhdXJvcmF1c2VyMDFAYXVyb3JhZmluYW5jZWludGVncmF0aW9uMDIub25taWNyb3NvZnQuY29tIiwidXBuIjoiYXVyb3JhdXNlcjAxQGF1cm9yYWZpbmFuY2VpbnRlZ3JhdGlvbjAyLm9ubWljcm9zb2Z0LmNvbSIsInV0aSI6ImFZMkdwM18xVWstRjRxRmtsaFFUQkEiLCJ2ZXIiOiIxLjAifQ.O-zTvqQfebyBfkX-ksbN95_2RwFBd062tRA52IB3orczQ5ub9xvI-VWbUlwkwzP3OBU4p4JdQdBM8R6Wc0UozK2H8K0b5ReEpLy2OS2ynEQnynuYht4MiBdz6AfcQ0TweILgA8FOYkdkSL0k9ZwCT4XPVbKLDjGadyI97UFvS20XxkEC30SDQQR7AtBa4qyWTfd9QePLAeuMvcghFaM7zvUHQZbNikm5AiiSTHP2map181AhmYlfO7Mv5q7sMb4mxkdIrzhnC64mwViuSijByebgs3MIQdmHJLF6hQY_NaKKExfPMKNZcViVKS_mCKKTQe-MN3SkF5wAKbW-7xGi6A [](http://example.com/codeflow?start=49&length=1606)

do we need a real jwt token? or just a string like  "faketoken"

Reply:
That's a security issue (even if not concerning as tokens have a short life) - needs to be fixed